SmartDeviceLink/SDLProtocolHeader.h

```diff
@@ -5,36 +5,36 @@
 
 
 typedef NS_ENUM(UInt8, SDLFrameType) {
-    SDLFrameType_Control = 0x00,
-    SDLFrameType_Single = 0x01,
-    SDLFrameType_First = 0x02,
-    SDLFrameType_Consecutive = 0x03
+    SDLFrameTypeControl = 0x00,
+    SDLFrameTypeSingle = 0x01,
+    SDLFrameTypeFirst = 0x02,
+    SDLFrameTypeConsecutive = 0x03
 };
 
 typedef NS_ENUM(UInt8, SDLServiceType) {
-    SDLServiceType_Control = 0x00,
-    SDLServiceType_RPC = 0x07,
-    SDLServiceType_Audio = 0x0A,
-    SDLServiceType_Video = 0x0B,
-    SDLServiceType_BulkData = 0x0F
+    SDLServiceTypeControl = 0x00,
+    SDLServiceTypeRPC NS_SWIFT_NAME(rpc) = 0x07,
+    SDLServiceTypeAudio = 0x0A,
+    SDLServiceTypeVideo = 0x0B,
+    SDLServiceTypeBulkData = 0x0F
 };
 
-typedef NS_ENUM(UInt8, SDLFrameData) {
-    SDLFrameData_Heartbeat = 0x00,
-    SDLFrameData_StartSession = 0x01,
-    SDLFrameData_StartSessionACK = 0x02,
-    SDLFrameData_StartSessionNACK = 0x03,
-    SDLFrameData_EndSession = 0x04,
-    SDLFrameData_EndSessionACK = 0x05,
-    SDLFrameData_EndSessionNACK = 0x06,
-    SDLFrameData_ServiceDataACK = 0xFE,
-    SDLFrameData_HeartbeatACK = 0xFF,
+typedef NS_ENUM(UInt8, SDLFrameInfo) {
+    SDLFrameInfoHeartbeat = 0x00,
+    SDLFrameInfoStartService = 0x01,
+    SDLFrameInfoStartServiceAck = 0x02,
```

Comment:
ACK and NACK should still be uppercase. They are based on the TCP ACK and NACK, which are uppercased.